docker-compose/.env

```diff
@@ -2,7 +2,7 @@ PG_MAX_CONNECTIONS=500
 CHAINLINK_VERSION="candidate-0.10.9-rc0.5ef19df076a4875c86455aa6db5ba56e514eedc9"
 CHAINLINK_IMAGE="public.ecr.aws/chainlink/chainlink"
 # Local Hardhat network defaults
-ETH_CHAIN_ID=31337
+ETH_CHAIN_ID=1337
```

Comment:
To be compatible with geth dev mode chainId must be 1337, sorry but this cannot be changed in geth https://github.com/ethereum/go-ethereum/issues/21120